server/routes/incidentRole/incidentRoleValidation.ts

```diff
@@ -0,0 +1,35 @@
+import { FormError } from '../../@types/template'
```

Comment:
Not sure this needs duplicating, but might do.. i need to check..

If it already exists then a rename is always better (if possible) as we keep the history then

Reply:
so for this one we do need two.  As this contains the validation for roles only, and the details version will have the roles part deleted once we remove that from the page